Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/ActorData.xml

```diff
@@ -32859,6 +32859,43 @@
     <CActorModel id="AP_CarrierHyperjumpTeleportOut" parent="ModelAnimationStyleOneShotNoOrphan">
         <Model value="AP_AirUnits_TeleportOut_COOP"/>
     </CActorModel>
+    <CActorModel id="AP_CarrierTaldarimLaserAttackAreaEffect" parent="ModelAnimationStyleOneShot">
+        <Model value="AP_HighArchonPsiStormImpact"/>
+        <On Terms="Effect.AP_CarrierTaldarimLaserDamageArea.Start" Send="Create"/>
+        <On Terms="ActorCreation" Send="AnimPlay Attack Attack"/>
+        <On Terms="ActorCreation" Send="TimerSet 0.550000 DelayDestroy"/>
+        <On Terms="ActorCreation" Send="SetScale 1.000000"/>
+        <On Terms="TimerExpired; TimerName DelayDestroy" Send="Destroy"/>
+        <On Terms="ActorCreation" Send="TimerSet 0.300000 DelayShrink"/>
+        <On Terms="TimerExpired; TimerName DelayShrink" Send="SetScale 0.400000"/>
+        <HostSiteOps Ops="SOpShadow"/>
+    </CActorModel>
+    <CActorAction id="AP_CarrierTaldarimLaserAttack" parent="GenericAttackBase">
+        <On Terms="ActorCreation" Send="SetTintColor 255,70,70"/>
+        <On Terms="Effect.AP_CarrierTaldarimLaserDamage.Start; At Caster; FromEffectTreeDescendant" Send="ActionDamage"/>
+        <On Terms="Effect.AP_CarrierTaldarimLaserCP.Start; At Caster" Send="Create"/>

```

Comment:
Missing `AP_CarrierTaldarimLaserCPSquadsight` (the attack visuals and sound won't be played if under Havoc's buff)

Reply:
This is a quick fix, commit pending on the other issue.